test/e2e/eksd.go

```diff
@@ -88,13 +88,6 @@ func EksdTerraformFlags(provider string) ([]string, error) {
 		flags := []string{
 			"-var", "initial_machinedeployment_replicas=0",
 			"-var", "static_workers_count=3",
-			// Amazon Linux 2 is currently not supported by machine-controller.
```

Comment:
It's supported as of 1.29.1 (https://github.com/kubermatic/machine-controller/pull/964)